engines/adl/detection.cpp

```diff
@@ -569,18 +569,18 @@ ADDetectedGames AdlMetaEngineDetection::detectGame(const Common::FSNode &parent,
 
 			game.matchedFiles[fileName] = filesProps[fileName];
 
-			if (game.hasUnknownFiles)
+			if (game.getHasUnknownFiles())
```

Comment:
I think this name is misleading. You're getting more than just `hasUnknownFiles()` flag now. And otherwise, having two verbs is misleading.  Maybe just turn it into a `hasUnknownFiles()` method?

Reply:
Frankly I was renaming only to catch all users. I'll rename back